soh/soh/z_message_OTR.cpp

```diff
@@ -187,4 +187,142 @@ extern "C" void OTRMessage_Init()
+                      "Si je me souviens bien, lors de&votre dernier transfert Warp,&votre solde était de %y{{playerBalance}}%w roupies."));
+    CustomMessageManager::Instance->CreateMessage(
+            customMessageTableID, TEXT_BANKER_WITHDRAWAL_AMOUNT,
+            CustomMessage("\x08Select Withdrawal Amount:&     rupees.&Set the amount with \xAA and&press \x9F to decide.\x0A",
+                        "\x08" "Auszahlungsbetrag:&     Rubine.&Wählen Sie den Betrag mit \xAA.&Drücken Sie \x9F, um zu entscheiden.\x0A",
+                        "\x08Sélectionnez le montant du retrait:&     Roupies.&Réglez le montant avec \xAA et&appuyez sur \x9F pour décider.\x0A"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_WITHDRAWAL_CONFIRM,
+        CustomMessage("Here are your rupees. Your new&balance is %y{{playerBalance}}%w rupees. See you&later!",
+                      "Hier sind deine Rupien. Ihr neues&Guthaben beträgt %y{{playerBalance}}%w Rupien.&Bis später!",
+                      "Voici vos roupies. Votre nouveau&solde est de %y{{playerBalance}}%w roupies. À plus&tard!"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_DEPOSIT_AMOUNT,
+        CustomMessage("\x08Select Deposit Amount:&     rupees.&Set the amount with \xAA and press&\x9F to decide.\x0A",
+                    "\x08" "Einzahlungsbetrag:&     Rubine.&Wählen Sie den Betrag mit \xAA.&Drücken Sie \x9F, um zu entscheiden.\x0A",
+                    "\x08Sélectionnez le montant du dépôt:&     Roupies.&Réglez le montant avec \xAA et&appuyez sur \x9F pour décider.\x0A"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_DEPOSIT_CONFIRM,
+        CustomMessage("You've deposited your rupees. Your&new balance is %y{{playerBalance}}%w rupees. See&you later!",
+                    "Sie haben Ihre Rupien eingezahlt.&Ihr neues Guthaben beträgt %y{{playexrBalance}}%w&Rupien. Bis später!",
+                    "Vous avez déposé vos roupies.&Votre nouveau solde est de %y{{playerBalance}}%w&roupies. À plus tard!"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_REWARD_WARP_TRANSFER_INTRO,
+        CustomMessage("Ho ho! Your savings have reached&%y200%w rupees! A fine achievement,&young spender!",
+                    "Ho ho! Ihre Ersparnisse haben %y200%w&Rupien erreicht! Eine tolle Leistung,&junger Spender!",
+                    "Ho ho! Vos économies ont atteint&%y200%w roupies! Une belle réussite,&jeune dépensier!"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_REWARD_WARP_TRANSFER_ITEM,
+        CustomMessage("To reward your thriftiness, I've got&a special trinket for you...The&%rPirate's Charm%w!",
+                    "Um deine Sparsamkeit zu belohnen,&habe ich ein besonderes&Schmuckstück für dich... den&%rPiratenzauber%w!",
```

Comment:
Considering the majoritarian usage of the polite form in the added dialogue lines, I would suggest to standardize the German dialogue in a consistent way.

`Um deine Sparsamkeit zu belohnen` &#8594; `Um Ihre Sparsamkeit zu belohnen`

`Schmuckstück für dich` &#8594; `Schmuckstück für Sie`